src/passwordHelpers.js

```diff
@@ -0,0 +1,17 @@
+import bcrypt from 'bcrypt-nodejs';
+
+export function comparePass(password, user) {
+  return new Promise((resolve, reject) => {
+    bcrypt.compare(password, user.password, (error, isAuth) => {
+      isAuth ? resolve(isAuth) : reject(error);
+    });
+  });
+}
+
+export function hashPass(password) {
+  return new Promise((resolve, reject) => {
+    bcrypt.hash(password, null, null, (error, hash) => {
```

Comment:
its sooooooooo simple. Just save your random string as a env var then concat that with the user id. and Boom! way more secure of a hash for such little work